### DISCUSSION TITLE
A403B304-9E2B-431D-B1B9-DDBD6C6B98AF

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#133] [General]
# A403B304-9E2B-431D-B1B9-DDBD6C6B98AF

Kepler-385, aka KOI-2433, is an f-type main sequence star located about 4670.31 lightyears from the solar system. the system has a minimum of 3 planets orbiting close to their host (c= 2.974 Re, 12.8 Me | b= 3.154 Re, 13.2 Me | d= 2.619 Re). planet d's mass is unknown currently. Additionally there are 3 other candidate planets still awaiting confirmation: two more mini-Neptunes and a super-Earth.

https://celestia.mobi/addon?item=A403B304-9E2B-431D-B1B9-DDBD6C6B98AF

**Comment by exocritic5457** [comment]
I don't think I could have seen this from any miles away. 7 hot planets in resonances, huddled around an F-type?? I need to start catching up haha
